stingray/lightcurve.py

```diff
@@ -458,7 +458,6 @@ def shift(self, time_shift):
 
         """
         new_lc = Lightcurve(self.time + time_shift, self.counts,
```

Comment:
Instead of eliminating `self.counts_err`, I would pass `self._counts_err` instead, which can be None or contain data, and doesn't trigger the `counts_err` calculation if None.